Server/sprawl_calculator/calculator/views.py

```diff
@@ -0,0 +1,13 @@
+from django.shortcuts import render
+from rest_framework.decorators import api_view
+# Create your views here.
+
+#Endpoint1_View
+@api_view(['PUT', 'GET'])
+def endpoint1(request):
+    raise NotImplementedError
+
+#Endpoint2_View
+@api_view(['PUT', 'GET'])
+def endpoint1(request):
```

Comment:
See above, this view cannot be `endpoint1` but should take in a uuid as a `processing_id` and should only have GET method, see [documentation](https://github.com/geodesignhub/urban-sprawl-calculation-service/blob/master/api/sprawl-calculation-service.yaml#L369)

Reply:
Both endpoints rename with suggested name.